src/ilasm/main.cpp

```diff
@@ -171,7 +171,7 @@ extern "C" int _cdecl wmain(int argc, __in WCHAR **argv)
       printf("\n/CLOCK          Measure and report compilation times");
 //      printf("\n/ERROR          Try to create .exe or .dll file despite errors reported");
 //      printf("\n       Warning! Results are unpredictable, use this option at your own risk!");
-      printf("\n/RESOURCE=<res_file>    Link the specified resource file (*.res) \n\t\t\tinto resulting .exe or .dll");
+      printf("\n/RESOURCE=<obj_file>    Link the specified object file (*.obj) created from a compiled resource file (*.res) \n\t\t\tinto resulting .exe or .dll");
```

Comment:
I would rather delete the `/RESOURCE` switch from this documentation so that the use of it does not spread. Leave it as undocumented switch that is used by a corefx build for now.

Reply:
Done!